seedcase_sprout/core/sprout_checks/data/get_pandera_checks.py

```diff
@@ -0,0 +1,152 @@
+import pandera.polars as pa
+from xmlschema.names import (
+    XSD_BASE64_BINARY,
+    XSD_DATE,
+    XSD_DATETIME,
+    XSD_DURATION,
+    XSD_GYEAR,
+    XSD_GYEAR_MONTH,
+    XSD_TIME,
+)
+
+from seedcase_sprout.core.properties import FieldProperties
+from seedcase_sprout.core.sprout_checks.data.data_type_checks import (
+    check_email,
+    check_geopoint,
+    check_json,
+    check_uuid,
+    check_xml_type,
+)
+
+# https://datapackage.org/standard/table-schema/#boolean
+BOOLEAN_VALUES = {"false", "False", "FALSE", "0", "true", "True", "TRUE", "1"}
+
+STRING_FORMAT_CHECKS = {
+    "email": pa.Check(
+        check_email,
+        element_wise=True,
+        error="Invalid email address.",
+    ),
+    "binary": pa.Check(
+        lambda value: check_xml_type(value, XSD_BASE64_BINARY),
+        element_wise=True,
+        error="Invalid data format. Expected a Base64-encoded string.",
```

Comment:
```suggestion
        error="The value {value} given isn't in a correct data format, we expect that it be Base64-encoded.",
```

I think we could make that language easier, I don't understand what the Base64 means and why it is important.

Reply:
You'd only see this error if you set the [format](https://datapackage.org/standard/table-schema/#string) of one of your fields as `binary`, so it would make more sense in that context, because you would expect the format to be validated.